definitions/checks/foreman_facts_names.rb

```diff
@@ -0,0 +1,25 @@
+class Checks::ForemanFactsNames < ForemanMaintain::Check
+  metadata do
```

Comment:
Is this check specific to Satellite not to Upstream?

If it is specific to Satellite then I would suggest to pass option :next_steps with knowledge base document link only when `feature(:downstream)` present.

Reply:
It's not downstream specific, we just happen to have a good content on the problem in a RH KBASE article.